jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -1227,6 +1195,11 @@ private static boolean isIncluded(HttpServletRequest request)
         return request.getAttribute(RequestDispatcher.INCLUDE_REQUEST_URI) != null;
     }
 
+    private static boolean isPathInfoOnly(HttpServletRequest req)
```

Comment:
I think `isDefaultServlet(Request)` or `isDefaultMapping(Request)` would be a clearer name.

Reply:
`isDefaultMapping()` sounds good but it requires inverting the logic; so
`if (isPathInfoOnly(req))` tests become `if (!isDefaultMapping(req))`.

Let me do it.